src/ResourceManager/KeyVault/Commands.KeyVault/Models/KeyAttributes.cs

```diff
@@ -65,7 +65,7 @@ public KeyAttributes()
 
         public DateTime? Updated { get; private set; }
 
-        public bool PurgeDisabled { get; private set; }
+        public string RecoveryLevel { get; private set; }
```

Comment:
This can't be a breaking change, since PurgeDisabled was never part of the service REST API spec. Yes, there is a publicly-released PowerShell out there which includes PurgeDisabled, but nobody should have taken a dependency on that since the feature was in preview.